packages/core-services/src/LocalBroker.ts

```diff
@@ -8,14 +9,25 @@ import type { EventSignatures } from './events/Events';
 import type { IBroker, IBrokerNode } from './types/IBroker';
 import type { ServiceClass, IServiceClass } from './types/ServiceClass';
 
+type ExtendedServiceClass = { instance: IServiceClass; dependencies: string[]; isStarted: boolean };
+
+const logger = new Logger('LocalBroker');
+
+const INTERVAL = 1000;
+const TIMEOUT = INTERVAL * 10;
+
 export class LocalBroker implements IBroker {
 	private started = false;
 
 	private methods = new Map<string, (...params: any) => any>();
 
 	private events = new EventEmitter();
 
-	private services = new Set<IServiceClass>();
+	private services = new Map<string, ExtendedServiceClass>();
+
+	private flattenedDependencies: Set<string> = new Set();
+
+	private defaultDependencies = ['settings'];
```

Comment:
Wouldn't License a default one too?

Reply:
no. for `LocalBroker` a license is not required (even though it will run anyways)